### DISCUSSION TITLE
Libre 1, NFC is not reacting

### DISCUSSION THREAD

**Discussion by Onweron** [NightscoutFoundation/xDrip#2313] [Q&A]
Hello there :)

pretty new to all the stuff regarding diabetes, apps, sensors, bridges etc.

Anyway, I have Libre 1, LibreLink is not available in my country. What I want is scanning my sensor directly via mobile phone.

And now I am totally confused. I followed instructions here: https://xdrip.readthedocs.io/en/latest/install/usethedoc/
I selected LibreAlarm App as a source, then I started new sensor (activated it via regular LibreReader before) and now I am stuck with Collecting Initial Readings window and... pretty much nothing. Sensor is reacting to regular reader, but not on phone both inside or outside of xDrip. NFC is turned on of course, settings in app should be OK aswell. I tried to restart the phone and reinstall xDrip multiple times, no change. Any tips for what I am doing wrong?
All help much appreciated.

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Can you explain what happens when you scan NFC with xDrip+?

**Reply by Onweron** [reply]
Well, nothing, it is almost like the NFC antenna is not working. But it is of course, I tried.

**Reply by psonnera** [reply]
Did you setup NFC parameters correctly? https://xdrip.readthedocs.io/en/latest/install/libreNFC/#enabling-nfc like both bottom options should be disabled. If you can scan with LibreLink make sure you don't let it interact with NFC. Are you confident you found the correct phone NFC antenna position and make sure you don't move the phone when resting it on the sensor?
Note that without a bridge (miaomiao/bubble) you will need to scan the sensor at least 3 times 5 minutes apart before you reach the calibration screen.

**Reply by Onweron** [reply]
Sorry for my late reply. As I said I dont have LibreLink so its no problem for me.
Settings for NFC are according to guidelines, I checked that multiple times.
Today I will change sensor so I will give it another try. Do I understand correctly that during those first 3 scans I will get nothing? No values, no vibrations, any sign that NFC scan proceeds succesfully?

**Reply by psonnera** [reply]
@Onweron no, you should have confirmation NFC scans correctly. Which phone is that?

**Reply by Onweron** [reply]
POCO F3

**Reply by Onweron** [reply]
OK, with new sensor, new behaviour. Now during NFC scan it is vibrating, also two small notifications popped up:
1) Scanning
2) NFC read timed out

**Reply by psonnera** [reply]
@Onweron I haven't found references to this phone in Libre Geeks Facebook group and that's not a good sign. I would recommend you try to find the vendor patched app to see if you can scan reliably NFC with it. https://xdrip.readthedocs.io/en/latest/troubleshoot/libre2/#patched-app

**Reply by Onweron** [reply]
Hello :)
just an ending... for some reason after attaching new sensor, everything started to work out perfectly both with direct scanning and using bridge (MiaoMiao), I have slight problem with bad calibration - for some reason it differs about 2 mmols over 8 hours but that is not really technical difficulty.
Many thanks for all your advice and patience.
